### ISSUE TITLE
Charts: default-theme is removed in V23

### ISSUE THREAD

**Issue by tomivirkki** [vaadin/docs#1186]
In Vaadin 14, `<vaadin-chart>`'s default theme was named ["vaadin-chart-default-theme"](https://github.com/vaadin/vaadin-charts/blob/41c10fea3905bd8130ef60406d685ed658a87a3e/theme/vaadin-chart-default-theme.html#L10). A theme name with "[component-tagname]-default-theme" has a [special purpose in themable-mixin](https://github.com/vaadin/web-components/blob/aa79465da8f3d9be92e22bdd04ef2a5fdb1d22a1/packages/vaadin-themable-mixin/vaadin-themable-mixin.js#L154); [it will get ignored in case there's a custom theme module targeting the same element](https://github.com/vaadin/web-components/blob/aa79465da8f3d9be92e22bdd04ef2a5fdb1d22a1/packages/vaadin-themable-mixin/vaadin-themable-mixin.js#L173) (a legacy feature that should probably be removed from themable-mixin).

This means that any application that needed to make theme customization on top of the charts default theme in a V14 app, had to add an additional include for the default theme to not have it discarded.

```html
<dom-module id="my-charts-theme" theme-for="vaadin-chart">
  <template>
    <style include="vaadin-chart-default-theme">
```

In V23, the charts default theme is renamed as ["vaadin-chart-base-theme"](https://github.com/vaadin/web-components/blob/aa79465da8f3d9be92e22bdd04ef2a5fdb1d22a1/packages/charts/theme/vaadin-chart-base-theme.js#L1026) so the special handling no longer takes place even if the app has custom styling for charts. Any existing includes for the "vaadin-chart-default-theme" should now be removed.

Also any explicit imports for `@vaadin/vaadin-charts/vaadin-chart-default-theme.js` will break the app and should be removed since the file no longer exists.

**Comment by tarekoraby** [comment]
Should this issue be closed by #1195?

**Comment by tomivirkki** [comment]
Yes it should, closing
